metabase/metabase
# Fix passing args in storybook for mantine components

Currently stories are implemented in a way that makes it impossible to re-configure arguments for any non-default story. This PR fixes the issue for all mantine components.

Before:

```
export const ReadOnly = args => <Template {...args} readOnly />

<Canvas>
  <Story name="Read only">
    {ReadOnly}
  </Story>
</Canvas>
```

After:

```
export const ReadOnly = Template.bind({});
ReadOnly.args = {
  readOnly: true
};

<Canvas>
  <Story name="Read only" args={ReadOnly.args}>
    {ReadOnly}
  </Story>
</Canvas>
```

How to verify:
- `yarn storybook`
- Compare with storybook in master for mantine components https://master--61d4abcd959e8c003aa51d4c.chromatic.com/
- In this branch you should be able to open any story with pre-configured args and change them later on
